code/modules/clothing/modular_armor/modular.dm

```diff
@@ -245,6 +245,51 @@
 	current_variant = variant
 	update_icon()
 
+
+/obj/item/clothing/suit/modular/pilot
```

Comment:
so upper level `/modular` is the jaeger exoskeleton. Since this is a direct subtype of that it will allow attaching leg, arm, and chest pieces.
make it a subtype of the xenonaut vests and then delete the copied vars. Basically look what vars are exactly the same on parent and delete them on child.